README.md

```diff
@@ -44,23 +44,36 @@ We have already implemented some pieces of this project:
 
 ## Wave 1
 
-Update `Square`, `Board` and `App.js` to respond to click events on the squares.  
+Update the `Board` component to render the grid of squares.  You will need to complete
 
-Each `Square` component should take 3 props.
+- `generateSquareComponents` in the `Board` component
+
+`App` should pass to `Board` a 2D array of JavaScript objects and Board should use that to render an array of `Square` components.
```

Comment:
Do we want to update the starter code to just create an array of 9 squares since we are relying on styling to actually dictate the number of squares per row anyway?

Reply:
It just seems a little funky to me to give them an array of arrays that they immediately flatten and do nothing meaningful with the nesting of the original array.